### DISCUSSION TITLE
Multiple watermeter sensors

### DISCUSSION THREAD

**Discussion by maciej871** [huizebruin/s0tool#226] [General]
Hi,
I built your water meter and it's great! But I have a question, how do I add two such sensors to Home Assistant? I have two meters: one for hot water and one for cold water. I would like to measure water consumption from individual meters and total. How do I add a second sensor and how do I configure it?
Thanks for your help.

**Comment by huizebruin** [comment]
Hello @maciej871 

How do I add a second sensor and how do I configure it?
Ad this moment thats not possible, and i would recommend to use then 2 wemos device's 


its very complex to make it for one user that one for 2 water meters.
